wp-content/themes/core/components/blocks/accordion/Accordion_Block_Controller.php

```diff
@@ -114,7 +118,10 @@ public function get_header_args(): array {
 
 	public function get_content_args(): array {
 		return [
-			Accordion_Controller::ROWS => $this->rows,
+			Accordion_Controller::ROWS            => $this->rows,
+			Accordion_Controller::CONTAINER_ATTRS => [
+				'data-scrollto' => $this->scroll_to,
+			],
```

Comment:
Right now, the necessary html attribute is added via the block, but this feature should also work at the component level (in cases where we use an accordion component outside the block context).

To make this work at the component level, you'll need to add a similar const, property, & attribute to the component controller. Then, rather than output the HTML attribute here, you'll want to do that inside the accordion component as one of it's conditional attributes instead of the block's.